modules/channel-smooch/src/backend/client.ts

```diff
@@ -84,20 +84,14 @@ export class SmoochClient {
       ? { channel: { smooch: { message: rawMessage } } }
       : {}
 
-    await this.bp.events.sendEvent(
-      this.bp.IO.Event({
-        botId: this.botId,
-        channel: 'smooch',
-        direction: 'incoming',
-        type: payload.type,
-        payload: { ...rawPayload, ...payload },
-        threadId: messagePayload.conversation._id,
```

Comment:
Even though we were using conversation ids here, we only send back messages in `sendMessage` using `event.target` (so appUser._id). In fact the `appUsers.sendMessage` function doesn't support providing a conversationId so I figured we might as well ignore conversation._id and just associate conversations with users using `conversations.recent()`

Reply:
You're saying there's no notion of conversation id in smooch ? 

Or is it that you just don't have it when you receive a message from smooch ?

Os is it that you don't have to provide it when you send a message to smooch ?

Reply:
Smooch has a notion of conversation id

We get this conversation id when we receive a message from smooch

BUT we don't use it we sending back a message to smooch.

So I figured we could just associate a conversation to a user for now. It's worth noting that smooch has a v2 version of their api that has come out, but if you look at the API version section of [this page](https://www.npmjs.com/package/smooch-core) for the `smooch-core` package, it doesn't seem supported at the moment. 

The v2 api seems to have a clearer support for multiple conversation per user, so if we ever update to that (meaning if smooch ever decides to update their `smooch-core` package) then we could support multiple conversations per user aswell with mappings